pkg/cmd/pulumi/packagecmd/package_extract_mapping.go

```diff
@@ -29,8 +29,8 @@ func newExtractMappingCommand() *cobra.Command {
 	var out string
 
 	cmd := &cobra.Command{
-		Use:   "get-mapping <key> <schema_source> [<provider key>]",
-		Args:  cobra.RangeArgs(2, 3),
+		Use:   "get-mapping <key> <schema_source> [<provider key>] [provider parameters]",
+		Args:  cobra.MinimumNArgs(2),
 		Short: "Get the mapping information for a given key from a package",
 		Long: `Get the mapping information for a given key from a package.
```

Comment:
Maybe add a note here that "" is valid for provider key and might need to be passed if wanting to use parameters.

Reply:
Good shout